tests/components/LoadingComponent.vue

```diff
@@ -0,0 +1,3 @@
+<template>
```

Comment:
Can we just use any sfc?

Reply:
Sure, I can use one of the existing ones